src/reducers/dashboard.js

```diff
@@ -51,6 +53,14 @@ const ACTION_HANDLERS = {
     ...state,
     allEvents: action.payload,
   }),
+  [SET_CALENDAR_TEAM_EVENTS]: (state, action) => ({
+    ...state,
+    allEvents: action.payload,
+  }),
+  [CLEAR_ALL_EVENTS]: (state, action) => ({
+    ...state,
+    allEvents: action.payload,
```

Comment:
referring to my comment at the top - could just do `allEvents: [],` - again - not a big deal

Reply:
Your right, to keep it consistent, i'll update this to be an empty array.